neo/UIntBase.cs

```diff
@@ -44,7 +43,10 @@ protected UIntBase(int bytes, byte[] value)
         /// </summary>
         void ISerializable.Deserialize(BinaryReader reader)
         {
-            reader.Read(data_bytes, 0, data_bytes.Length);
+            if (reader.Read(data_bytes, 0, data_bytes.Length) != data_bytes.Length)
```

Comment:
Sounds redudant but maybe looks necessary, right?

Reply:
How could you read something and it be different than the size of it?

Reply:
With less data

Reply:
But how less, since `data_byte.Lenght` was fixed?
I will leave this for you guys, too deep to me...aheuaheauea jajaja
I will talk to Igor.

Reply:
@vncoelho  it reads up to "length", it doesn't mean it will always read that length (if it has only one byte, like Shargon mentioned)